src/imitation/scripts/expert_demos.py

```diff
@@ -40,7 +43,7 @@ def rollouts_and_policy(
 
   policy_save_interval: int = -1,
   policy_save_final: bool = True,
-) -> None:
+) -> dict:
```

Comment:
:+1: this is useful even without `multi_expert_demos` (Sacred will save it so it's a useful summary)